METADATA_SUPPORT.md

```diff
@@ -489,6 +489,7 @@ v55 introduces the following new types.  Here's their current level of support
 |ConvReasonReportDefinition|❌|Not supported, but support could be added|
 |ConvReasonReportSegmentDef|❌|Not supported, but support could be added|
 |CustomAddressFieldSettings|✅||
+|DataImportManagementSettings|✅||
```

Comment:
@mshanemc Guess this got auto-updated? Should I keep/revert this?

Reply:
I will revert this change just to be safe

Reply:
yeah I saw this on one of my PRs as well (not this specific change, but the same pattern).  We should make a note to understand why it does this at some point :)

Reply:
there's an automated script that will pull the latest metadata types and update the registry, I think it's a `pre-push` hook

Reply:
Thanks, not reverting this add then as it seems to be have been added by design